### PR TITLE
Clean up vision tests

### PR DESCRIPTION
**Pull request by NielsRogge**
# What does this PR do?

This is a follow-up of #16799. It took me way too long to realize I don't need to overwrite `test_attention_outputs` and `test_hidden_states_outputs` as I can just set the `seq_length` attribute of the ModelTester. 😂

### CODE REVIEW

File: tests/beit/test_modeling_beit.py
Comment (ydshieh) [inline tests/beit/test_modeling_beit.py:201]: Very nice!

**Review by ydshieh — APPROVED** [review]
LGTM, thank you for this clean-up!
(as well as for the work on the TF/Flax files ❤️ )

BTW, we should still have at least one review from some core maintainers.

**Review by sgugger — APPROVED** [review]
Very nice cleanup!

**Pull request merged**
